src/Akka.Persistence.MongoDb.Tests/Akka.Persistence.MongoDb.Tests.csproj

```diff
@@ -7,11 +7,14 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(TestSdkVersion)" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="$(XunitVersion)" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
```

Comment:
Same with this - using the `$(XunitVersion)` variable from `common.props` make it possible to sync all XUnit dependencies all in one go as well, unless the two versions are diverged